scripts/_compile/index.js

```diff
@@ -75,12 +75,28 @@ async function compileLess () {
  * values are CSS stylesheet strings
  */
 async function compileScss () {
+    let rendered;
+
     // SASS -> CSS (full)
-    let rendered = await SASS.renderSync({
-        ...CONFIG.sass,
-        file: `${CONFIG.sourceDir}/scss/index.scss`,
-        outputStyle: 'expanded',
-    });
+    try {
+        rendered = await SASS.renderSync({
+            ...CONFIG.sass,
+            file: `${CONFIG.sourceDir}/scss/index.scss`,
+            outputStyle: 'expanded',
+        });
+    } catch (err) {
+        // failed render
+        console.error(err.formatted);
+
+        let cssMessage = `/* ${err.formatted} */`;
+        return {
+            css: cssMessage,
+            minified: cssMessage,
+        };
```

Comment:
## BEFORE
Console output:
![CleanShot 2019-09-30 at 16 32 32@2x](https://user-images.githubusercontent.com/545605/65918750-08989280-e3a0-11e9-87e7-2043496deae3.png)


## AFTER
Example console error:
![CleanShot 2019-09-30 at 16 28 52@2x](https://user-images.githubusercontent.com/545605/65918608-ac357300-e39f-11e9-9100-d8a839a2f16b.png)


Example CSS output:
![CleanShot 2019-09-30 at 16 29 20@2x](https://user-images.githubusercontent.com/545605/65918623-b35c8100-e39f-11e9-8044-2c18029315ce.png)

Reply:
@CITguy, the formatted error comments you added are much more useful. 💯